diff --git a/stratosphere/loader/source/ldr_config.cpp b/stratosphere/loader/source/ldr_config.cpp
index e0602f0..e1323b2 100644
--- a/stratosphere/loader/source/ldr_config.cpp
+++ b/stratosphere/loader/source/ldr_config.cpp
@@ -136,7 +136,7 @@ bool LoadConfigFromFile(FILE* filePtr, LoaderConfig& outConfig)
 const LoaderConfig& LoaderConfig::tryLoadingFromFile()
 {
     if (!configLoadedFromFile)
-        configLoadedFromFile = LoadConfigFromFile(fopen("sdmc:/rajnx_ipl.ini", "r"), *this);
+        configLoadedFromFile = LoadConfigFromFile(fopen("sdmc:/cfw/hbl.ini", "r"), *this);
 
     return *this;
 }
diff --git a/stratosphere/loader/source/ldr_content_management.cpp b/stratosphere/loader/source/ldr_content_management.cpp
index a6658f9..402f181 100644
--- a/stratosphere/loader/source/ldr_content_management.cpp
+++ b/stratosphere/loader/source/ldr_content_management.cpp
@@ -40,7 +40,7 @@ Result ContentManagement::MountCode(u64 tid, FsStorageId sid) {
         const char* customNspPath = nullptr;
         const auto& ldrConfig = g_ldrConfig.tryLoadingFromFile();        
         if (tid == ldrConfig.wantedTitleId && ldrConfig.shouldRedirectBasedOnKeys())
-            customNspPath = "@Sdcard:/RajNX/hbl.nsp";
+            customNspPath = "@Sdcard:/cfw/hbl.nsp";
 
         if (R_SUCCEEDED(MountCodeNspOnSd(tid, customNspPath)))
             return 0;
